src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+function successHandler(text) {
+  const message = document.createElement('div');
+
+  message.className = 'message';
+  message.textContent = text;
+  document.body.append(message);
+}
+
+function errorHandler() {
+  const message = document.createElement('div');
+
+  message.className = 'message error-message';
+  message.textContent = 'Promise was rejected!';
+  document.body.append(message);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+promise1.then(successHandler).catch(errorHandler);
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error);
```

Comment:
```suggestion
    reject(new Error('Promise was rejected!'));
```
if you pass an error in the reject function you should pass its instance (i.e. create new Error)
In this case, you can read error.message in errorHandler